src/data.js

```diff
@@ -67,24 +67,44 @@ module.exports = [
     ],
   },
   {
-    name: 'Chandu J S',
+    name: 'Jerry Shi',
```

Comment:
Did you intend to update someone else's entry?

Reply:
woops ! My bad.